### ISSUE TITLE
Build failed for iOS simulator on 2.0.0-pre3

### ISSUE THREAD

**Issue by Soarc** [TorbenK/TK.CustomMap#279]
I'm building my app with TK.CustomMap dependency and getting this error:
 ```
clang : error : linker command failed with exit code 1 (use -v to see invocation) [/s/src/MyApp/MyApp.csproj]
  error MT5209 : Native linking error : warning: ignoring file Native References/ClusterKit.framework/ClusterKit, missing required architecture x86_64 in file Native References/ClusterKit.framework/ClusterKit (2 slices) [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: CKBottomCluster. The symbol '_OBJC_CLASS_$_CKBottomCluster' could not be found in any of the libraries or frameworks linked with your application. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: CKCentroidCluster. The symbol '_OBJC_CLASS_$_CKCentroidCluster' could not be found in any of the libraries or frameworks linked with your application. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: CKCluster. The symbol '_OBJC_CLASS_$_CKCluster' could not be found in any of the libraries or frameworks linked with your application. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: CKClusterAlgorithm. The symbol '_OBJC_CLASS_$_CKClusterAlgorithm' could not be found in any of the libraries or frameworks linked with your application. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: CKClusterAnimation. The symbol '_OBJC_CLASS_$_CKClusterAnimation' could not be found in any of the libraries or frameworks linked with your application. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: CKClusterManager. The symbol '_OBJC_CLASS_$_CKClusterManager' could not be found in any of the libraries or frameworks linked with your application. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: CKGridBasedAlgorithm. The symbol '_OBJC_CLASS_$_CKGridBasedAlgorithm' could not be found in any of the libraries or frameworks linked with your application. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: CKNearestCentroidCluster. The symbol '_OBJC_CLASS_$_CKNearestCentroidCluster' could not be found in any of the libraries or frameworks linked with your application. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: CKNonHierarchicalDistanceBasedAlgorithm. The symbol '_OBJC_CLASS_$_CKNonHierarchicalDistanceBasedAlgorithm' could not be found in any of the libraries or frameworks linked with your application. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: CKQuadTree. The symbol '_OBJC_CLASS_$_CKQuadTree' could not be found in any of the libraries or frameworks linked with your application. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5214: Native linking failed, undefined symbol: _kCKMarginFactorWorld. This symbol was referenced by the managed member Xamarin.iOS.ClusterKit.Constants.kCKMarginFactorWorld. Please verify that all the necessary frameworks have been referenced and native libraries linked. [/s/src/MyApp/MyApp.csproj]
  MTOUCH : error MT5202: Native linking failed. Please review the build log. [/s/src/MyApp/MyApp.csproj]
 
```

**Comment by TorbenK** [comment]
I probably need to rebuild the clusterkits binding to support x86 arcitecture. Apple does not support it anymore, so you can also just switch you compile target arcitecture.

**Comment by Soarc** [comment]
There is no option to turn off x86 architecture when building for simulation. At least in GUI.

![image](https://user-images.githubusercontent.com/3385495/33718912-78a16b42-db78-11e7-8f13-428dfcc7128e.png)

I think, the solution would be to rebuild clusterkits with [fat frameworks](https://developer.xamarin.com/guides/cross-platform/macios/native-references/#Thin_vs._Fat_Frameworks) support.

**Comment by TorbenK** [comment]
try not to use the simulator.

**Comment by nbsoftware** [comment]
Damn. I just stumbled on this issue. And I just can confirm it.

This really hinders productivity, not being able to build and run the app on the simulator anymore.

I'm not interested (yet) in the Cluster functionality (which by the way is enabled by default - I had to disable it), alas I need a couple of fixes that are in that release (MapReady & the offset in FitMapRegionToPositions).

Is there any way around this?

**Comment by geoffwc** [comment]
I managed to build a fat binary to replace the framework in the CK binding lib. Works fine with no changes to binding code. (It's a .a file so change .txt to .a). Just delete the ClusterKit native ref and add this one instead.
[libClusterKit.txt](https://github.com/TorbenK/TK.CustomMap/files/1631761/libClusterKit.txt)

**Comment by TorbenK** [comment]
fix included in 2.0
